rmw_fastrtps_cpp/CMakeLists.txt

```diff
@@ -38,10 +38,8 @@ find_package(rmw_dds_common REQUIRED)
 find_package(rmw_fastrtps_shared_cpp REQUIRED)
 find_package(tracetools REQUIRED)
 
-find_package(fastrtps_cmake_module REQUIRED)
```

Comment:
Do we need to keep `fastrtps_cmake_module` around at all, or should we look at removing that package entirely?

Reply:
This might be removed.

Reply:
See https://github.com/ros2/rosidl_typesupport_fastrtps/pull/120 for this